api/layer/util_test.go

```diff
@@ -158,7 +158,7 @@ func Test_objectInfoFromMeta(t *testing.T) {
 
 	for _, tc := range cases {
 		t.Run(tc.name, func(t *testing.T) {
-			info := objectInfoFromMeta(bkt, tc.object, tc.prefix, tc.delimiter)
+			info := processObjectInfoName(objectInfoFromMeta(bkt, tc.object), tc.prefix, tc.delimiter)
```

Comment:
It doesn't matter a lot, but it'd be more consistent to rename `func Test_objectInfoFromMeta`